app/experimenter/kinto/serializers.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from rest_framework import serializers
+
+from experimenter.experiments.models import Experiment, ExperimentVariant
+
+
+class ExperimentRapidBranchesSerializer(serializers.ModelSerializer):
+    value = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ExperimentVariant
+        fields = ("slug", "ratio", "value")
+
+    def get_value(self, obj):
+        # placeholder value
+        return None
```

Comment:
Is this supposed to be an empty object?  I guess if it validates with the external schema then this is fine?

Reply:
i changed it to None, since @k88hudson  mentioned that A/A was gonna send null?